src/utils/supergraph_config.rs

```diff
@@ -484,71 +308,7 @@ mod test_get_supergraph_config {
                 });
                 when.method(httpmock::Method::POST)
                     .path("/")
-                    .json_body_obj(&json!({
-                        "query": indoc!{
-                            r#"
-                        query SubgraphFetchAllQuery($graph_ref: ID!) {
-                          variant(ref: $graph_ref) {
-                            __typename
-                            ... on GraphVariant {
-                              subgraphs {
-                                name
-                                url
-                                activePartialSchema {
-                                  sdl
-                                }
-                              }
-                            }
-                          }
-                        }
-                        "#
-                        },
-                        "variables": {
-                            "graph_ref": graphref_raw,
-                        },
-                        "operationName": "SubgraphFetchAllQuery"
-                    }));
-                then.status(200)
-                    .header("content-type", "application/json")
-                    .json_body(body);
-            });
-
-            server.mock(|when, then| {
-                let body = json!({
-                  "data": {
-                    "graph": {
-                      "variant": {
-                        "subgraphs": [
-                          {
-                            "name": name
-                          }
-                        ]
-                      }
-                    }
-                  }
-                });
-                when.method(httpmock::Method::POST)
-                    .path("/")
-                    .json_body_obj(&json!({
-                        "query": indoc!{
-                          r#"
-                      query IsFederatedGraph($graph_id: ID!, $variant: String!) {
-                        graph(id: $graph_id) {
-                          variant(name: $variant) {
-                            subgraphs {
-                              name
-                            }
-                          }
-                        }
-                      }
-                      "#
-                        },
-                        "variables": {
-                            "graph_id": name,
-                            "variant": "current"
-                        },
-                        "operationName": "IsFederatedGraph"
-                    }));
```

Comment:
This bit of code is now unnecessary because of Glasser's refactor a while back. Therefore it's just adding lines for no good reason